events/wmt.md

```diff
@@ -0,0 +1,99 @@
+name: WMT
+---
+
+**WMT** is the main event for machine translation and machine translation research.  The conference is held annually in connection with larger conferences on natural language processing.
+
+> The conference aims to bring together academic scientists, researchers and industry representatives to exchange and share their experiences and research results. WMT plays a key role for the entire industry of computational linguistics and machine translation.
+
+In 2006, the first **Workshop on Machine Translation** was held at the NAACL (North American Chapter of the Association for Computational Linguistics Annual Meeting).
+
+In 2016, with the rise of [neural machine translation](approaches/neural-machine-translation.md), WMT became a conference of its own.  The **Conference on Machine Translation** is still mainly known as *WMT*.
+
+Universities, research laboratories and big technology companies consistently participate in the conference and are represented in the organising committee.
+
+## Shared tasks
+
+WMT hosts shared tasks competitions prior to the conference. During the shared tasks, researchers tackle specific challenges from a list of themes. Researchers then compare their techniques against the techniques of other researchers in the field.
+
+During the main conference, researchers present the results of the shared tasks and winners are announced.
+
+WMT started in 2006 with a *translation* task. In the following editions, WMT included themes on all aspects of machine translation, corpus preparation, training, and evaluation.
+
+
+#### Translation tasks
+
+- General machine translation task (former News task)
+- Medical text translation task
+- Biomedical translation task
+- Pronoun translation task
+- Bilingual document alignment
+- Multimodal translation task
+- Similar language translation task
+- Unsupervised and very low resource translation task
+- Lifelong learning in machine translation task
+- Chat translation task
+- Multilingual low-resource translation task for Indo-European languages
+- Life-long learning in machine translation task
+- Machine translation using terminologies task
+- Sign language translation task
+- Robustness translation task
+- Triangual machine translation task
+- Large-scale multilingual machine translation task
+
+#### Evaluation tasks
+
+- Metrics task
+- Quality estimation task
+- Tuning task
+
+#### Other tasks
+
+- Automatic post-editing task
+- Parallel corpus filtering task
+- Task on training of neural machine translation
+- Task on bandit learning for machine translation
+
+The results published from WMT shared tasks represent benchmarks for the technology.
+
+### Long-time organisers
+
+- [Philipp Koehn](community/people/philipp-koehn.md)
+- Barry Haddow
+- Loïc Barrault
+- Ondřej Bojar
+- Marco Turchi
+- Martin Popel
+- Matt Post
+- Rajen Chatterjee
+- Christof Monz
+- Matteo Negri
+- Marco Turchi
+- Matthias Huck
+- Christian Federmann
+- Christof Monz
+
+## Editions
+
```

Comment:
I might just move this to the top section, so that we don't have to give it a name.